### ISSUE TITLE
Bug: After manual update from .69 to .99 ferdi only shows blank window

### ISSUE THREAD

**Issue by Sarcas666** [ferdium/ferdium-app#478]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.0.0.nightly.99

### What Operating System are you using?

Windows

### Operating System Version

Window 11

### What arch are you using?

x64

### Last Known Working Ferdium version

6.0.0-nightly.96, but not anymore.

### Expected Behavior

Expected Ferdium to start and show all installed services and messages

### Actual Behavior

Ferdium only shows a coloured empty window. No services, no menus.

### Steps to reproduce

I double-click the icon, or choose 'open' from contextual menu.

### Additional information

I had version .96 running without problems but noticed there was an update. Auto-update gave me an error. I forgot which one. I blamed my firewall/pi-hole so downloaded the latest windows auto install full package for .99. After installation, it only showed the empty window. 
I tried uninstalling and reinstalling, and rebooting, but the problem persists. I uninstalled and went to .98, with the same results. .97 , .96 and the latest beta; same results. 
Unsure how to proceed.
I found issue #230 (closed) but I did not understand the solution, if any.

**Comment by SpecialAro** [comment]
Hello @Sarcas666 , thank you for raising this issue.

If I understand correctly you where on .69 and upgraded to .99. In .69 we didn't had any code sign in-place and so much time as passed that the changes probably did break something on your end.

Are you running Ferdium with a local account or connected to the Ferdium Server?

If you can, please uninstall ALL instances of Ferdium and proceed to delete your `%APPDATA%/Roaming/Ferdium` too, and then install again the latest nightly. Also, make sure you're installing the version specific to your arch (x64, arm64, i32)

**Comment by Sarcas666** [comment]
Hi @SpecialAro , thanks for the reply. I made a stupid typo; I was updating from .96 to .99. 'Check for updates' gave me an error ('An error occurred. (Check the console for more details)'. 

I uninstalled Ferdium and removed the directory you mentioned, and reinstalled the latest nightly for my arch. Things are working again; I logged into my Ferdium account and all is well. 
I tried the 'Check for updates' again and got the same error message. A rare flash of insight made me enable 'Include pre-releases'. That error is gone now; I guess the only problem was being on a nightly (.96) while somehow having the  'Include pre-releases' option turned off, and manually installing .99 made things somehow worse. But all is solved now, for me. 
Thank you for your time.
